stats_prometheus.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"github.com/falcosecurity/falcosidekick/types"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+func getInitPromStats() *types.PromStatistics {
+	promStats = &types.PromStatistics{
+		Falco:   getFalcoNewCounterVec(),
+		Inputs:  getInputNewCounterVec(),
+		Outputs: getOutputNewCounterVec(),
+	}
+	return promStats
+}
+
+func getInputNewCounterVec() *prometheus.CounterVec {
+	return promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "falcosidekick_inputs",
+		},
+		[]string{"source", "status"},
+	)
+}
+
+func getOutputNewCounterVec() *prometheus.CounterVec {
+	return promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "falcosidekick_outputs",
+		},
+		[]string{"destination", "status"},
+	)
+}
+
+func getFalcoNewCounterVec() *prometheus.CounterVec {
+	return promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "falco_events",
+		},
+		[]string{
+			"rule",
+			"priority",
+			"k8s_ns_name",
+			"k8s_pod_name",
+		},
```

Comment:
Are you planning to use the [same naming convention of falco-exporter](https://github.com/falcosecurity/falco-exporter/blob/master/pkg/exporter/exporter.go#L15-L25)?
It's already so, and it would be great if we can keep this kinda compatibility in the future too.

Reply:
yes, I kept the same labels, except `hostname`, because it's only available when gRPC is used

Reply:
Not sure if adding those two empty fields is better. It could be already ok so  :+1: 

PS
btw great job, bravo! :clap:

Reply:
thanks. the PR could be merge now I guess